state/protocol/badger/snapshot.go

```diff
@@ -416,23 +417,21 @@ func (q *EpochQuery) NextUnsafe() (protocol.TentativeEpoch, error) {
 	if err != nil {
 		return nil, fmt.Errorf("could not get protocol state snapshot at block %x: %w", q.snap.blockID, err)
 	}
-	phase := epochState.EpochPhase()
 	entry := epochState.Entry()
```

Comment:
since that call doesn't error, you could inline it in https://github.com/onflow/flow-go/blob/5d0a86e2126b36ac650f04a2d10d60e395fced26/state/protocol/badger/snapshot.go#L428 (for code compactness)